hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CacheRemovePartitionLostListenerMessageTask.java

```diff
@@ -38,8 +37,8 @@ public CacheRemovePartitionLostListenerMessageTask(ClientMessage clientMessage,
     @Override
     protected Object call() {
         ICacheService service = getService(CacheService.SERVICE_NAME);
```

Comment:
I think accessing service name using different classes is confusing. Can't we use `ICacheService.SERVICE_NAME` also here?